lesson_1/task_2.py

```diff
@@ -0,0 +1,15 @@
+# Напишите код, который запрашивает число и сообщает является ли оно простым или составным. Используйте правило для проверки: “Число является простым, если делится нацело только на единицу и на себя”. Сделайте ограничение на ввод отрицательных чисел и чисел больше 100 тысяч.
+
+def is_simple(number):
+    for i in range(2, int(number**0.5) + 1):
+        if number % i == 0:
+            return False
+    return True
+
+if __name__ == '__main__':
+    number = int(input("Введите целое положительное число меньше 100 000:"))
+    if number < 0 or number > 100000:
+        print("Неправильный ввод")
+    else:
+        print(f"{number} является простым числом") if is_simple(number) else print(f"{number} не является простым числом") 
```

Comment:
Вот ещё вариант с более короткой записью условия.
```suggestion
    if 0 <= number <= 100000:
        print(f"{number} является простым числом") if is_simple(number) else print(f"{number} не является простым числом") 
    else:
        print("Неправильный ввод")
```